src/model/constraints/behavior_constraints.py

```diff
@@ -0,0 +1,37 @@
+from abc import abstractmethod
+from src.model.constraints.constraint import Constraint
+
+class Behavior_Con(Constraint):
```

Comment:
Zhekai, I have a structural comment. Since `Behavior_Con` is an abstract constraint (it canot be directly instantiated), I think it would be better to put it in the same file as the [Constraint](https://github.com/ArenBabikian/concretize/blob/d0a7aa6c88f081c43e1802b3b40c6110a2432bf8/src/model/constraints/constraint.py) file. The same applies to the `Static_Con` class